src/domain/profile/Profile.tsx

```diff
@@ -49,14 +48,15 @@ const Profile = () => {
               <h1>
                 {data.myProfile.firstName} {data.myProfile.lastName}
               </h1>
-              <Button
-                className={styles.editProfile}
-                ariaLabel={t('profile.edit.button.text')}
-                onClick={() => setIsOpen(true)}
-              >
-                <span>{t('profile.edit.button.text')}</span>
-                <Icon src={settingsIcon} alt="" />
-              </Button>
+              <div className={styles.editProfile}>
+                <button
```

Comment:
Did you mean to use the basic HTML button?

Reply:
Yes. I'll add a comment as to why, it is because HDS adds some whitespace that breaks the layout and is hard/impossible to remove AFAIK